doc/user_guide.rst

```diff
@@ -336,15 +336,28 @@ connect(parameters...)
             *This must be a keyword parameter.*
 
          charset
-            If present, the connection character set will be changed
-            to this character set, if they are not equal. Support for
-            changing the character set requires MySQL-4.1 and later
-            server; if the server is too old, UnsupportedError will be
-            raised. This option implies use_unicode=True, but you can
-            override this with use_unicode=False, though you probably
-            shouldn't.
-
-            If not present, the default character set is used.
+            If supplied, the connection character set will be changed
+            to this character set.
```

Comment:
What `changed` means? This is constructor. No previous charaset.

Reply:
This is the language from the previous doc. I did not write it or change it.

> If present, the connection character set will be changed to this character set, if they are not equal.

Reply:
Sorry.